package-lock.json

```diff
@@ -5667,18 +5667,27 @@
       "integrity": "sha1-vsEf3IOp/aBjQBIQ5AF2wwJNFWc="
     },
     "scandal": {
-      "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/scandal/-/scandal-3.1.0.tgz",
-      "integrity": "sha1-m0AkuXxxm74lAIzAm6rHn7tdNQE=",
+      "version": "3.2.0",
+      "resolved": "https://registry.npmjs.org/scandal/-/scandal-3.2.0.tgz",
+      "integrity": "sha512-kXICe3ygxwyyO3Ur+k49UzQlu8yrdQgzD03eMgV8sMWDom9q4qpEvZuQRUcbyAujC1TpISPRUPoirOIO1bRxcQ==",
       "requires": {
         "argparse": "^1.0.2",
-        "git-utils": "^5.0.0",
+        "git-utils": "^5.6.0",
         "isbinaryfile": "^2.0.4",
         "minimatch": "^2.0.9",
         "split": "^1.0.0",
         "temp": "^0.8.3"
       },
       "dependencies": {
+        "git-utils": {
```

Comment:
This is odd...seems like it should be deduped with the top-level version as the two version numbers are the same.

Reply:
good point, I'm gonna try to dedupe then in a followup PR